.github/workflows/lead_provider_openapi_check.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       DB_PORT: 5432
       DATABASE_URL: postgres://postgres:postgres@localhost:5432
       SECRET_KEY_BASE: abc123
+      NODE_OPTIONS: --openssl-legacy-provider
```

Comment:
As it's not obvious, this looks like it's needed [because of a bug in WebPack](https://github.com/webpack/webpack/issues/14532#issuecomment-947515866).

Reply:
👍 Yeah took a good while to figure this out as the build was broken without it